src/protocol/damage.rs

```diff
@@ -88,6 +88,17 @@ impl From<u8> for ReportLevel {
         Self(value)
     }
 }
+impl std::fmt::Debug for ReportLevel  {
+    fn fmt(&self, fmt: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let variants = [
+            (Self::RAW_RECTANGLES.0.into(), "RAW_RECTANGLES", "RawRectangles"),
+            (Self::DELTA_RECTANGLES.0.into(), "DELTA_RECTANGLES", "DeltaRectangles"),
+            (Self::BOUNDING_BOX.0.into(), "BOUNDING_BOX", "BoundingBox"),
+            (Self::NON_EMPTY.0.into(), "NON_EMPTY", "NonEmpty"),
+        ];
+        pretty_print_enum(fmt, self.0.into(), &variants)
```

Comment:
Why not a `match` (or two depending on `fmt`)?

Reply:
Bad reason: I started writing the `bit` variant and then made the other version similar.

Slightly better reason, but I am not sure if this is true: #488. I hope that we only end up with one implementation of the code in the binary that just searches through some lookup table. With a `match`, I fear that the binary size would be larger. However, I didn't test this and it is just a theory.

Also: Two `match`es would mean more lines of code. :-)